src/Teams/Controllers/ManageTeamsController.cs

```diff
@@ -1,21 +1,36 @@
+using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using Microsoft.Extensions.DependencyInjection;
 using System.Diagnostics;
+using System.Linq;
+using Teams.Data;
 using Teams.Models;
+using Teams.Security;
+using Teams.Services;
 
 namespace Teams.Controllers
 {
     public class ManageTeamsController : Controller
     {
+        private readonly IManageTeamsService _teamsService;
+        public ManageTeamsController(  IManageTeamsService teamsService)
```

Comment:
please add empty line before constructor and remove not needed space before parameter